QUICK-START.md

```diff
@@ -99,4 +99,4 @@ populated `CumulocityConnectorConfig.json` from earlier.
     1. Located in the `starting-files` folder
 
 Once uploaded, restart the Ewon device and the Ewon Flexy Cumulocity Connector will start up
-automatically (using the uploaded `jvmrun` file).
\ No newline at end of file
+automatically (using the uploaded `jvmrun` file).
```

Comment:
whitespace change on last line of file

Reply:
I think a newline got added or deleted. just fix that and it's fine from me

Reply:
Oh no - I just merged it before your comment.

Reply:
I'm not sure why this change is present - I used the GitHub online editor.

Reply:
that's okay, it's very minor